idseq_pipeline/commands/host_filtering_functions.py

```diff
@@ -75,6 +75,8 @@
 
 # pipeline configuration
 LZW_FRACTION_CUTOFF = 0.45
+MAX_INPUT_LINES = 3*10**8 # must be multiple of 4 (fastq)
```

Comment:
maybe " 3*(10**8) "

Reply:
that does look nicer

Reply:
I've seen this more customarily written as `30*1000*1000`.